src/common/helpers/proxy/setup-proxy.js

```diff
@@ -9,9 +9,9 @@ const logger = createLogger()
 /**
  * If HTTP_PROXY is set setupProxy() will enable it globally
  * for a number of http clients.
- * Node Fetch will still need to pass a ProxyAgent in on each call.
+ * Node Fetch and similar clients will still need to pass a ProxyAgent in on each call.
```

Comment:
We don't make any API calls from this repo so this comment isn't really relevant.  However, as it's part of the CDP template and we don't know the future of the service, have left it in.

Just modifying it because teams may use Wreck, Axios etc in favour of Node Fetch.